modules/nf-core/fibertoolsrs/extract/meta.yml

```diff
@@ -0,0 +1,56 @@
+name: "fibertoolsrs_extract"
+description: Extract Fiber-seq data into plain text files
```

Comment:
This is a bed file presumably? I know those are plain text, but they are well-known as a file type.